### DISCUSSION TITLE
Absence de son lorsque downloadhelper télécharge une video

### DISCUSSION THREAD

**Discussion by oliversailles** [aclap-dev/video-downloadhelper#1010] [Help and Q&A]
### Your question

Bonjour, 
J'ai acheté une licence car je souhaitais obtenir un audio à partir d'une video. En lecture sur le site la video a bien du son mais une fois téléchargée plus de son. 
Il s'agit d'une série de 9 videos et seulement 2 vidéos téléchargées ont du son 
Merci pour votre aide 
https://p1ps.cloud.panopto.eu/Panopto/Pages/Viewer.aspx?id=3bf29692-bc7f-4fcc-b091-b0ba00cd7af3&instance=Cours-cavej

### Addon & CoApp version and details

https://p1ps.cloud.panopto.eu/Panopto/Pages/Viewer.aspx?id=3bf29692-bc7f-4fcc-b091-b0ba00cd7af3&instance=Cours-cavej

**Comment by Mattmjs** [comment]
Can you click the 3 dot button next to download and go down to Details. A page will open with all the information.
Copy this as text ( not an image ) and paste the information here for further help.

**Comment by oliversailles** [comment]
Version 9.0.2.12 (stable), target mozilla, locale fr.Appli compagnon installée (v2.0.19) Re-testerC:\Program Files (x86)\DownloadHelper CoApp\vdhcoapp.exeLicence vérifiée ***@***.***)envoyé : 16 juillet 2024 à 09:36de : Mattmjs ***@***.***>à : aclap-dev/video-downloadhelper ***@***.***>cc : oliversailles ***@***.***>, Author ***@***.***>objet : Re: [aclap-dev/video-downloadhelper] Absence de son lorsque downloadhelper télécharge une video (Discussion #1010) Can you click the 3 dot button next to download and go down to Details. A page will open with all the information. Copy this as text ( not an image ) and paste the information here for further help.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by Mattmjs** [reply]
You've provided your version information, but more information is needed about the video which you've provided the link to.
No one can see it because it's not public. If you visit your problem video, provide the details about it.
Example:

![16-7-24-22 00](https://github.com/user-attachments/assets/262b676c-e6a9-4e88-8523-2a14f69e0556)

**Comment by oliversailles** [comment]
Tab5 (incognito: false)TitleRegroupement N°6 du 1 Décembre 2023 de 19h à 20h30Page URLhttps://p1ps.cloud.panopto.eu/Panopto/Pages/Viewer.aspx?id=0b67450f-f56c-441b-995c-b0ce00ef8d2e&instance=Cours-cavej with 10 headersThumbnailhttps://d2hpwsdp0ihr0w.cloudfront.net/sessions/71ed7d0f-b880-4662-b985-b0ce00ef8d26/uploadedimages/b98f2039-288d-4186-8ed2-b0cb0153f617.pngLow Qualityfalse#0 main urlhttps://d2hpwsdp0ihr0w.cloudfront.net/sessions/71ed7d0f-b880-4662-b985-b0ce00ef8d26/f6680cde-658f-493a-a94b-b0ce00ef8d2f.object.hls/master.m3u8?InvocationID=81cf429e-6e43-ef11-aa0f-0699e0158e14&tid=00000000-0000-0000-0000-000000000000&StreamID=f6680cde-658f-493a-a94b-b0ce00ef8d2f&ServerName=p1ps.cloud.panopto.eu#0 video componenthttps://d2hpwsdp0ihr0w.cloudfront.net/sessions/71ed7d0f-b880-4662-b985-b0ce00ef8d26/f6680cde-658f-493a-a94b-b0ce00ef8d2f.object.hls/1029687/index.m3u8#0 audio componentnone#0 core mediaMp4 b:Hls p:hls 7325.16s#0 videoH264 at 3728394, fps: ?, q:720, {“width”:1936,“height”:1068}envoyé : 16 juillet 2024 à 14:06de : Mattmjs ***@***.***>à : aclap-dev/video-downloadhelper ***@***.***>cc : oliversailles ***@***.***>, Author ***@***.***>objet : Re: [aclap-dev/video-downloadhelper] Absence de son lorsque downloadhelper télécharge une video (Discussion #1010) You've provided your version information, but more information is needed about the video which you've provided the link to. No one can see it because it's not public. If you visit your problem video, provide the details about it. Example:16-7-24-22.00.png (view on web)—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by Mattmjs** [reply]
That helped but this only confirms it's downloading the video without audio as you explained up the top.
The main url only has information for a video, maybe the video has another url for the audio which VDH is not seeing.
You might have to record the network activity and attach it here as a **.zip** file so it can be investigated further.
Head back to your video and open the developer tools / Network Monitor by pressing **Ctrl + Shift + E**, 
make sure it is empty if not click the trash can on the left hand side. Then reload the page, once it stops click gear button on
the right side and Save all as HAR.


![3-10-23](https://github.com/user-attachments/assets/88d0acd2-e3bf-4087-a5e7-de92a33f7ac0)

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.  Same as Matt.

I extracted the URL for the HLS master manifest that @oliversailles posted above:

`https://d2hpwsdp0ihr0w.cloudfront.net/sessions/71ed7d0f-b880-4662-b985-b0ce00ef8d26/f6680cde-658f-493a-a94b-b0ce00ef8d2f.object.hls/master.m3u8?InvocationID=81cf429e-6e43-ef11-aa0f-0699e0158e14&tid=00000000-0000-0000-0000-000000000000&StreamID=f6680cde-658f-493a-a94b-b0ce00ef8d2f&ServerName=p1ps.cloud.panopto.eu`

I thought that was posted so many hours ago that it might have expired.  But no, ffprobe gave me a report.

[ffprobe.txt](https://github.com/user-attachments/files/16258747/ffprobe.txt)

There is no audio with this video.  If you play the video in the web page, do you hear audio?  When you do press the play button in the player on the web page, does new content appear in the VDH menu?  My suspicion is that new content would be either (a) the audio track or (b) a whole new video that actually has audio.  But this video is a silent movie.  VDH is not failing.  It is downloading what is there.  What is there is not what you want.  What you want has audio but that's not what's being presented.  If there is something that has audio, you need to find it.  We can't help because the site is behind user authentication.  Only you can find the media you want.
